01-tree-specification.bs

```diff
@@ -118,21 +117,15 @@ In one tree, completeness MUST be guaranteed, unless indicated otherwise (as is
 
 # Initialization # {#init}
 
-A client SHOULD be initiated using a URL.
-The client MUST dereference the URL, which will result in a set of [[!rdf-concepts]] triples or quads.
-When the URL after all redirects, is used in a triple `?c tree:view <> .`, a client MUST assume the URL after redirects is an identifier of the intended root node of the collection in `?c`.
+The main TREE specification specifies a TREE-based client MUST be initiated either by using the IRI of the `tree:Collection`, or by using the IRI of the root `tree:Node`.
```

Comment:
Why not just the following? It feels weird to read this in the actual "main TREE specification".
```suggestion
A TREE-based client MUST be initiated either by using the IRI of the `tree:Collection`, or by using the IRI of the root `tree:Node`.
```